lib/broccoli/ember-app-with-packages.js

```diff
@@ -15,11 +15,7 @@ function EmberAppWithPackages(defaults, options) {
       // TODO: re-enable jshint once it's actually working fine, for now it just slows down the build
       // and running `ember test` it only jshints boot, but not the packages. We'll rely an npm script
       // for now
-      hinting: false,
-      fingerprint: {
-        // Disabling here since we do it at the end for *all* the assets
-        enabled: false,
```

Comment:
I think that removing this may impact the build size since we would be fingerprinting twice. 

Was there an issue with this?